Apps/ValidationTests/Shared/TestUtils.h

```diff
@@ -28,18 +37,17 @@ namespace Babylon
 
             Napi::Function func = ParentT::DefineClass(
                 env,
-                "TestUtils",
+                "TestUtilsClass",
                 {
-                    ParentT::StaticMethod("exit", &TestUtils::Exit),
-                    ParentT::StaticMethod("updateSize", &TestUtils::UpdateSize),
-                    ParentT::StaticMethod("setTitle", &TestUtils::SetTitle),
-                    ParentT::StaticMethod("writePNG", &TestUtils::WritePNG),
+                    ParentT::InstanceMethod("exit", &TestUtils::Exit),
+                    ParentT::InstanceMethod("updateSize", &TestUtils::UpdateSize),
+                    ParentT::InstanceMethod("setTitle", &TestUtils::SetTitle),
+                    ParentT::InstanceMethod("writePNG", &TestUtils::WritePNG),
+                    ParentT::InstanceMethod("decodeImage", &TestUtils::DecodeImage),
+                    ParentT::InstanceMethod("getImageData", &TestUtils::GetImageData),
+                    ParentT::InstanceMethod("getWorkingDirectory", &TestUtils::GetWorkingDirectory),
```

Comment:
getWorkingDirectory is the easiest way I found to get it working for now as I need absolute path for Linux